ci/cloudbuild/triggers/clang-tidy-compute.yaml

```diff
@@ -13,9 +13,10 @@ sourceToBuild:
   repoType: GITHUB
   uri: https://github.com/googleapis/google-cloud-cpp
 substitutions:
-  _BUILD_NAME: clang-tidy-compute
+  _BUILD_NAME: clang-tidy
```

Comment:
comment: It should be fine to reuse this output directory.

https://github.com/googleapis/google-cloud-cpp/blob/d807b786a8300f128d782fb7e5e2342c41725247/ci/cloudbuild/build.sh#L297

Reply:
Thanks for catching that, fixed now.

Reply:
So it wasn't fine?

Reply:
It does not work right if you try to compile two shards locally.